Chef.html

```diff
@@ -16,8 +16,39 @@ <h1> The Lamb Sauce </h1>
 <h1> Chef Baxter </h1>
 
 <p> He starts out as a chef in a restaurant, but he later opens his own restaurant, The Chill Grill. His biggest rival is Leonard Stevenson. In Cory in the House, he gets a job as the President's chef, and it is unknown what became of his restaurant. His catchphrases are "Here comes the pain!" and "I'll go pack!"</p>
-<img src="https://vignette.wikia.nocookie.net/cory/images/9/98/Victor.jpg/revision/latest?cb=20090528021740" alt="Italian Trulli" class="baxter">
 
 
+<div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+  <ol class="carousel-indicators">
+    <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+    <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+    <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+  </ol>
```

Comment:
You should also use the bootstrap grid: container - row - col to structure your page. The concept about the lamb sauce is nice but, it doesnt look very good...